about.html

```diff
@@ -0,0 +1,135 @@
+                        <a href="#" class="nav-link">Sponsor</a>
+                    </li>
+                    <li class="nav-li">
+                        <button href="#" class="nav-btn">ITech campain</button>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+    <section class="section-1 about-section1">
+        <h2 class="page-greeting">"Hello sharing world"</h2>
+        <h1 class="page-title about-title">Technology Innovation Summit 2023</h1>
+    <p class="about-description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptas, error mollitia? Ab ipsa,
+        a vitae maxime ducimus omnis voluptas voluptate alias? Veritatis maxime necessitatibus, aut vel nihil accusamus
+        obcaecati et placeat eligendi sit qui voluptatum perferendis, beatae odit deserunt eum pariatur, hic reiciendis
+        sint? Suscipit inventore dignissimos nesciunt minus quisquam?</p>
+    <p class="about-contact">Please contact us per Email for any further question about Technology Innovation summit 2023</p>
+    <a href="#" class="about-email">myemail@gmail.com</a>
+    </section>
+    <section class="about-section__2">
+        <h2 class="about-section2-title">Technolody Innovation Global Summit 2023</h2>
+        <hr class="underline">
+        <p class="page-text">The logo of Technology Innovation Summit 2023 was decided through the logo competition from 8 Jun to 7 July</p>
+        <p class="about-contact-desktop">After online public voting and Selection Committee's judging process, the logo from
+            Salomon Tshobohwa.
+            After consultation with Mr Agrawal, we optimized the prize-winning-logo for this year summit as following:
+        </p>
+        <div class="logo-div">
+            <img src="icons/page-logo.png" alt="Innovation technology summit" class="about-logo">
+        </div>
+    </section>
+    <section class="about-section__3">
+        <h2 class="about-section3-title">See the past Technology Innovation Summit</h2>
+        <hr class="underline">
+        <p class="page-text">Take a look at the two previous I Tech Summit which took place in Kigali and Kampala</p>
+        <div class="previous-summits">
+            <div class="previous-summit">
+                <img src="images/tech-summit-kigali.jpg" alt="I Tech Summit in Kigali" class="previous-summit-image">
+                <div class="previous-summit-overlay">
+                    <h4>2019</h4>
+                    <p class="previous-summit-text">I Tech Summit 2019 in Kigali</p>
+                </div>
+            </div>
+            <div class="previous-summit">
+                <img src="images/tech-summit-Kampala.jpg" alt="I Tech Summit in Kampala" class="previous-summit-image">
+                <div class="previous-summit-overlay">
+                    <h4>2021</h4>
+                    <p class="previous-summit-text">I Tech Summit 2021 in Kampala</p>
+                </div>
```

Comment:
- [ ] Kindly note that in the `featured` speakers section, the general background should look like `d3d3d3` to match the original design.

![image](https://user-images.githubusercontent.com/98453979/219662372-b0414500-2592-4900-9058-68fbe8d70c1f.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/219662526-82eb247a-f304-4840-821d-6352de1c1a65.png)